src/middlewared/middlewared/service.py

```diff
@@ -297,7 +297,7 @@ def __init__(self, middleware, parts):
 
         for part in self.parts:
             for name in dir(part):
-                if name.startswith('_'):
+                if name.startswith('__'):
```

Comment:
We won't allow exposed method names to start with `_`, why?

Reply:
`_get_instance` is a good example here. Compound services can't use it

Reply:
Then we need a public alias for _get_instance that will eventually replace it

Reply:
We still need to revert this